github/resource_github_release.go

```diff
@@ -82,6 +82,56 @@ func resourceGithubRelease() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"release_id": {
```

Comment:
How come this additional schema is added in this PR? Shouldn't there be a corresponding docs change for this as well?

Reply:
They are already in the docs as attribute reference, they were missing from the schema I guess because it was allowed in older tf sdk versions but as of today the resource is breaking and this is fixing it. The attributes are all computed and so only attribute reference, not parameters.

Reply:
Ahh got it, thanks!